app/src/main/kotlin/co/netguru/baby/monitor/client/feature/settings/ClientSettingsFragment.kt

```diff
@@ -16,9 +19,16 @@ import co.netguru.baby.monitor.client.common.base.BaseFragment
 import co.netguru.baby.monitor.client.common.extensions.*
 import co.netguru.baby.monitor.client.feature.client.home.ClientHomeViewModel
 import co.netguru.baby.monitor.client.feature.voiceAnalysis.VoiceAnalysisOption
+import io.reactivex.Observable
+import io.reactivex.Single
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.rxkotlin.plusAssign
+import io.reactivex.schedulers.Schedulers
 import kotlinx.android.synthetic.main.fragment_client_settings.*
 import pl.aprilapps.easyphotopicker.EasyImage
 import java.io.File
+import java.util.concurrent.TimeUnit
 import javax.inject.Inject
 
 class ClientSettingsFragment : BaseFragment() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/c7380d74fc32b4ddceef02c8d9a561008c986f4f/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/settings/ClientSettingsFragment.kt#L34"></span>Class ‘ClientSettingsFragment’ with ‘20’ functions detected. Defined threshold inside classes is set to ‘15’</td>
    </tr>
  </tbody>
</table>